deploy/helm/sumologic/templates/events/otelcol/statefulset.yaml

```
@@ -100,6 +100,7 @@ spec:
       containers:
       - name: otelcol
         image: {{ .Values.otelevents.image.repository }}:{{ .Values.otelevents.image.tag }}
+        imagePullSecrets: {{ .Values.otelevents.image.pullSecrets }}
```

For consistency, this should be 
```suggestion
{{- if .Values.sumologic.pullSecrets }}
        imagePullSecrets:
{{ toYaml .Values.sumologic.pullSecrets | indent 10 }}
{{ end }}
```
unless you specifically want a different secret here?